kotlin-grpc/src/main/resources/META-INF/services/com.linecorp.armeria.server.annotation.DelegatingResponseConverterFunctionProvider

```diff
@@ -0,0 +1 @@
+com.linecorp.armeria.internal.common.kotlin.FlowResponseConverterFunctionProvider
```

Comment:
Question: Why do we need this provider? `armeria-kotlin` which this module depends on provides the file already.

Reply:
Ref: https://github.com/line/armeria/pull/4724#discussion_r1127589412